doc/source/flexibility/binlogging_replication_improvements.rst

```diff
@@ -0,0 +1,196 @@
+.. _binlogging_replication_improvements:
+
+=======================================
+Binlogging and replication improvements
+=======================================
+
+Due to continuous development, |Percona Server| incorporated a number of
+improvements related to replication and binary logs handling. This resulted in
+replication specifics, which distinguish it from |MySQL|.
+
+Temporary tables and mixed logging format
+=========================================
+
+Summary of the fix:
+*******************
+
+As soon as some statement involving temporary table was met when using mixed
+binlog format, |MySQL| was switching to row-based logging of all statements the
+end of the session or until all temporary tables used in this session are
+dropped. It is inconvenient in case of long lasting connections, including
+replication-related ones. |Percona Server| fixes the situation by switching
+between statement-based and row-based logging as and when necessary.
+
+Version Specific Information
+****************************
+
+  * :rn:`5.6.21-70.1`
```

Comment:
Version specific info always concerns only the current version of the manual. There it should be e.g. "5.7.10-1 feature ported from 5.6"